lib/debug/console.rb

```diff
@@ -56,9 +56,9 @@ def readline_setup prompt
         Reline.prompt_proc = -> args, *kw do
           case state = parse_input(args.first, commands)
           when nil, :command
-            [prompt, prompt]
+            [prompt] * args.size
           when :ruby
-            [prompt.sub('rdbg'){colorize('ruby', [:RED])}] * 2
+            [prompt.sub('rdbg'){colorize('ruby', [:RED])}] * args.size
           end
```

Comment:
Should we add `* args.size` here instead?

Reply:
updated